README.md

```diff
@@ -1,4 +1,6 @@
 # Nexus Mod Manager
+[![Build status](https://ci.appveyor.com/api/projects/status/mgt1kuc1a0eqju7m?svg=true)](https://ci.appveyor.com/project/Nexus-Mods/nexus-mod-manager)
```

Comment:
API key should be replaced with token from official Nexus-Mods appveyor account.